python_scripts/parameter_tuning_nested.py

```diff
@@ -215,7 +215,7 @@
 # ```
 #
 # However, this evaluation only provides us a single point estimate of the
-# generalization performance. As recall at the beginning of this notebook, it is
+# generalization performance. As you recall at the beginning of this notebook, it is
```

Comment:
Hi @psteinb thanks for the PR! Maybe this wording is a bit better (it may need formatting the line length though):
```suggestion
# generalization performance. As you recall from the beginning of this notebook, it is
```

Reply:
Totally, thanks for lending a second pair of eyes.